NEWS.md

```diff
@@ -14,6 +14,8 @@
 
 ## Fixes
 
+* Improvements to setting column widths. Previously values set by `set_col_widths()` were a little off. This has been fixed. [350](https://github.com/JanMarvin/openxlsx2/pull/350)
```

Comment:
Should note that it still may not be exact on Windows?

Reply:
Yes, will do

Reply:
Updated NEWS.